lib/src/analyzers/lint_analyzer/rules/rules_list/prefer_define_hero_tag/visitor.dart

```diff
@@ -0,0 +1,31 @@
+part of 'prefer_define_hero_tag_rule.dart';
+
+const _floatingActionButtonClassName = 'FloatingActionButton';
```

Comment:
Is the rule really limited only to check for `FloatingActionButton`? There are no other widgets with `heroTag`?

Reply:
Hero Widget ?

Reply:
Hero widget has required field `tag` and does not need this rule.
On the other hand, there are two more widgets with field `heroTag`: `CupertinoNavigationBar` and `CupertinoSliverNavigationBar`.

Reply:
Maybe we should check all of them with this rule?

Reply:
Yes, I added check for them.